docs/learn-teleport.md

```diff
@@ -0,0 +1,41 @@
+---
+id: learn-teleport
+title: Teleporting Assets To Other Parachains
+sidebar_label: Teleporting Assets
+---
+
+One of the main properties Polkadot and Kusama brings to ecosystem is blockchain interoperability.
+Teleportation is the process of moving assets, such as coins, tokens, or NTFs, to other chains
+(parachains) to use them as you would any other asset native to that chain. Interoperability is
+possible through [XCMP] and [SPREE modules], which together ensure that that assets are not lost or
+duplicated across multiple chain.
+
+## How to Teleport
+
+Teleportation can be done through the [PolkadotJS Apps] interface or through the
+`xcmPallet.teleportAssets()` extrinsic. For this example, we will be using the PolkadotJS interface.
```

Comment:
Just because the example isn't introduced yet, better if it's "The following example below uses..."